src/core/providers/ogr/qgsogrprovider.cpp

```diff
@@ -1452,13 +1452,72 @@ QVariant QgsOgrProvider::defaultValue( int fieldId ) const
   else if ( defaultVal == QStringLiteral( "CURRENT_TIME" ) )
     resultVar = QTime::currentTime();
 
+  // Get next sequence value for sqlite in case we are inside a transaction
+  if ( mOgrOrigLayer &&
+       mTransaction &&
+       mDefaultValues.value( fieldId, QString() ) == tr( "Autogenerate" ) &&
```

Comment:
Is "Autogenerate" translated? Isn't that coming from sqlite itself?

Reply:
nope

Reply:
fingers crossed we have a consistent translation of this magic value everywhere.

Reply:
it's the same for OGR provider, I think I homogenized spatialite provider taking OGR as the source of truth a while go ... I think it's ok to translate it because it's exposed to the users but yeah: we definitely need to pay attention to translate it all the times we refer to it in the code.

Reply:
I agree, let's keep it like this for now.

To make the link: https://github.com/qgis/QGIS/issues/36396